temel_bilgiler/javascript_temelleri/fundamentals-3.md

```diff
@@ -1,20 +1,20 @@
-### Introduction
+### Giriş
 
-Things are about to get _really_ exciting. So far you have been writing an impressive amount of code to solve various problems, but that code has not been as useful as it could be. Imagine taking one of your scripts and bundling it into a little package that you could use over and over again without having to rewrite or change the code. That's the power of functions, and they're used _constantly_ in JavaScript.
+İşler _gerçekten_ heyecan verici bir hal almak üzere. Şimdiye kadar çeşitli problemleri çözmek için etkileyici miktarda kod yazdınız, ancak bu kod çok kadar kullanışlı değildi. Kodlarınızdan birini aldığınızı ve kodu yeniden yazmak veya değiştirmek zorunda kalmadan tekrar tekrar kullanabileceğiniz küçük bir paket haline getirdiğinizi hayal edin. İşte fonksiyonların gücü budur ve JavaScript'te _sürekli_ kullanılırlar.
 
-### Lesson Overview
+### Derse Genel Bakış
 
-This section contains a general overview of topics that you will learn in this lesson.
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içermektedir.
 
-- How to define and invoke different kinds of functions.
-- How to use the return value.
-- What function scope is.
+- Farklı fonksiyon türlerini nasıl tanımlanır ve çağrılır?
+- Dönüş değeri nasıl kullanılır?
+- Fonksiyon kapsamı nedir?
 
-### Functions
+### Fonksiyonlar
 
-1. [This lengthy MDN article](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions) is a good place to start. Don't worry as there may be some functions that can be beyond the reach of this particular lesson, but do pay special attention to the sections on 'Function Scope'. Scope is a topic that commonly trips up both beginner and intermediate coders, so it pays to spend some time with it upfront. Note that this article also has its own exercises attached, which you should **not** do, as they involve knowledge we haven't touched yet.
-2. Read this article about [return values](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Return_values).
-3. Let's discuss parameters and arguments in the context of the following example function:
+1. [Bu uzun İngilizce dilindeki MDN makalesi](<https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions>) başlangıç için iyi bir noktadır. Bu dersin kapsamı dışında kalabilecek bazı fonksiyonlar da bulunduğu için endişelenmeyin,'Fonksiyon Kapsamı' ile ilgili bölümlere  odaklanın. Kapsam (scope), genellikle hem yeni başlayan hem de orta düzey yazılımcıları uğraştıran bir konudur, bu nedenle şimdiden bu konuya biraz zaman ayırmak faydalı olacaktır. Bu makale, henüz değinmediğimiz konuları da içerdiği için henüz **yapmamanız** gereken alıştırmaları da içerebilir.
```

Comment:
fix: Some links throughout the page has extra `<>` wrappers `[Bu uzun İngilizce dilindeki MDN makalesi](<https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions>)` , it may be occuring due to some kind of auto-formatting.